core/app/models/spree/order.rb

```diff
@@ -37,7 +37,8 @@ class Order < ActiveRecord::Base
 
     alias_attribute :ship_total, :shipment_total
 
-    has_many :state_changes, as: :stateful
+    belongs_to :store, class_name: 'Spree::Store'
```

Comment:
This addition was part of a merge conflict I had to resolve when cherry-picking. I'm a little perplexed on why it's actually an addition as it was present in the parent branch.